gimp/pictonode-gimp-plugin/GIMP 2.99/sandbox/plugin_ui_sandbox.py

```diff
@@ -0,0 +1,148 @@
+# To run, make sure ../scripts/GtkNodes/output exists by:
+
+# cd to ../scripts/GtkNodes
+# run python3 setup_gtknodes.py
+
+import os
+import sys
+
+if sys.platform == "linux":
+    os.environ['GI_TYPELIB_PATH'] = '/usr/lib/x86_64-linux-gnu/girepository-1.0'
```

Comment:
Isn't there a more elegant solution to this in `GIRepository`?

Reply:
There is, but was under the impression that `GI_TYPELIB_PATH` env variable needed to be set to bootstrap it when not initialized by GIMP, but looks like that path is an element of default paths for the PyGObject module and isn't even needed.

Reply:
#46